runtime/opt/taupage/init.d/10-prepare-disks.py

```diff
@@ -213,20 +243,20 @@ def should_format_volume(region, partition, erase_on_boot):
     #
     if erase_on_boot is None:
         ec2 = ec2_client(region)
-        volumes = list(ec2.get_all_volumes(filters={
+        volumes = list(get_all_volumes(ec2, filters={
```

Comment:
It should still work like this, but for consistency maybe remove `filters=` part?